packages/fcl/package.json

```diff
@@ -58,5 +69,25 @@
     "@onflow/util-template": "^1.1.0",
     "@onflow/util-uid": "^1.1.0",
     "cross-fetch": "^3.1.6"
+  },
+  "peerDependencies": {
+    "@react-native-async-storage/async-storage": "^1.18.0",
```

Comment:
This version is newer than version supported in the latest expo sdk (I had to force install for this version to be compatible). Maybe we can go a little older?

Reply:
I dialed down the dependencies versions